packages/contentstack-migrate-rte/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@contentstack/cli-cm-migrate-rte",
   "description": "Contentstack CLI plugin to migrate HTML RTE to JSON RTE",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "author": "contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-command": "^1.2.0",
+    "@contentstack/cli-utilities": "^1.2.0",
     "@contentstack/json-rte-serializer": "^2.0.1",
     "@contentstack/management": "^1.6.1",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K we can remove this package if not used directly

Reply:
@antonyagustine removed @oclif/core dependency from package.json